src/shared/dropdown-list-v2/types.ts

```diff
@@ -40,6 +40,7 @@ export interface DropdownSearchProps<T> {
     enableSearch?: boolean | undefined;
     /** If specified, the default no results display will not be rendered */
     hideNoResultsDisplay?: boolean | undefined;
+    noResultsDesc?: string | undefined;
```

Comment:
this can also support JSX elements for custom descriptions, like bold text etc.

```suggestion
    noResultsDesc?: React.ReactNode | undefined;
```